client/components/admin/AddEmployee/AddEmployee.tsx

```diff
@@ -35,41 +38,51 @@ const AddEmployee = (props: AddEmployeeProps) => {
     pin: pin,
     first_name: first_name,
     last_name: last_name,
-    phone: phone,
+    phone: cleanedPhone,
     email: email,
     hourly_rate: hourly_rate
   };
 
-  // const handleSubmit2 = (event: React.FormEvent<HTMLFormElement>) => {
-  //   const form = event.currentTarget;
-  //   if (form.checkValidity() === false) {
-  //     event.preventDefault();
-  //     event.stopPropagation();
-  //   }
+   // Update the cleaned phone state when the input value changes
+    const handlePhoneChange = (value: string) => {
+    const cleaned = cleanPhoneNumber(value);
+    setPhone(value);
+    setCleanedPhone(cleaned);
+    console.log('cleaned:', cleaned)
+  };
 
-    // This syntax, replacing the above, prevented an error from occurring in employeeController middleware.  
+   // This syntax, replacing the above, prevented an error from occurring in employeeController middleware.  
     const handleSubmit2 = (event: React.FormEvent<HTMLFormElement>) => {
       event.preventDefault();
       event.stopPropagation();
     
       const form = event.currentTarget;
-      if (form.checkValidity() === false) {
+  
+      // send a window.alert() if the phone number is invalid
+      if (cleanedPhone === null) {
+        window.alert('You have entered an invalid phone number: please check your input and try again.');
```

Comment:
Can we add some [form validation](https://react-bootstrap.netlify.app/docs/forms/validation/) here instead of the window.alert?